beacon-chain/sync/validate_aggregate_proof.go

```diff
@@ -275,7 +275,26 @@ func (s *Service) validateIndexInCommittee(ctx context.Context, bs state.ReadOnl
 		return result, err
 	}
 
-	committee, result, err := s.validateBitLength(ctx, bs, a.GetData().Slot, a.GetData().CommitteeIndex, a.GetAggregationBits())
+	var committeeIndex primitives.CommitteeIndex
+	if a.Version() >= version.Electra {
+		a, ok := a.(*ethpb.AttestationElectra)
+		// This will never fail in practice because we asserted the version
+		if !ok {
+			err := fmt.Errorf("attestation has wrong type (expected %T, got %T)", &ethpb.AttestationElectra{}, a)
+			tracing.AnnotateError(span, err)
+			return pubsub.ValidationIgnore, err
+		}
+		var validationRes pubsub.ValidationResult
+		committeeIndex, validationRes, err = validateCommitteeIndexElectra(ctx, a)
```

Comment:
I see validateCommitteeIndex above on line 273, is there some benefit to validating here vs just merging this with validate committeeIndex above and having the electra validation in there?